main.go

```diff
@@ -88,16 +89,14 @@ func main() {
 
 	metricsClient := metrics.New()
 
-	resource.RuntimeScheme = mgr.GetScheme()
-
-	creator := resource.NewCreator(metricsClient, clients.Interface)
+	creator := resource.NewCreator(clients.Interface, metricsClient, poll.New(), scheme)
```

Comment:
Won't it make a code a little bit confusing? Maybe we can change all the packages interfaces initializations in one final PRs, once all the packages has been converted into typed interface?

Reply:
I tried adding a global variable to `pkg/resource`, but not initializing it from `main` proved difficult. Let's leave it like this for now, `pkg/helmer` refactoring is next in line anyway.